src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -595,22 +870,18 @@ end
 		itemId = "Red Paintball Gun"
 	})
 	```
-	
+
 	@param data PurchaseEvent
-	@return { [any]: any }
+	@return Promise
 ]=]
-function AnalyticsService:LogPurchase(data: PurchaseEvent): { [any]: any }
+function AnalyticsService:LogPurchase(data: PurchaseEvent)
```

Comment:
```suggestion
function AnalyticsService:LogPurchase(data: PurchaseEvent): {[any]: any}
```
Outside of a custom defined promise type, {[any]: any} is the closest to a return type for a promise in my opinion.